# Geometrical features message
std_msgs/Header header
mesh_msgs/MeshGeometryStamped mesh
geometry_msgs/Point[] normals
geometry_msgs/Point[] baricenters
traversability_mesh_package/Proximals neighbors
